src/components/Login/Login.scss

```diff
@@ -20,24 +20,25 @@ input {
   width: 100%;
   padding: 0.5em;
   margin-bottom: 0.5em;
-  border: 1px solid #ccc;
+  border: 2px solid #ccc;
   border-radius: 5px;
 }
 
 .error {
   color: red;
 }
 
-button {
+.button-login {
   width: 100%;
   padding: 0.7em;
-  background-color: #007BFF;
+  background-color: #313237;
   color: white;
   border: none;
   border-radius: 5px;
   cursor: pointer;
+  margin-bottom: 10px
 }
 
 button:hover {
-  background-color: #0056b3;
+  background-color: #313237;
 }
```

Comment:
use of tag here changes button hover in other places, channge it to class. Also you should change text color on hover so text would not dissapear